eos/rare-b-decays/charm-loops-analytic.cc

```diff
@@ -0,0 +1,219 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2022 Viktor Kuschke
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#include <eos/maths/power-of.hh>
+#include <eos/maths/polylog.hh>
+#include <eos/maths/multiplepolylog_li22.hh>
+#include <eos/rare-b-decays/charm-loops.hh>
+#include <eos/rare-b-decays/long-distance.hh>
+#include <eos/utils/exception.hh>
+#include <eos/utils/log.hh>
+#include <eos/utils/stringify.hh>
+
+#include <cmath>
+#include <complex>
+#include <cstring>
+#include <vector>
+
+#include <gsl/gsl_spline.h>
+#include <gsl/gsl_sf_dilog.h>
```

Comment:
Can you remove the include statements that are removes in the following commit? Since we will merge all the commits it probably doesn't matter...